server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -146,7 +147,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion INITIAL_ELASTICSEARCH_8_18_2 = def(8_840_0_04);
     public static final TransportVersion INITIAL_ELASTICSEARCH_8_18_3 = def(8_840_0_05);
     public static final TransportVersion INITIAL_ELASTICSEARCH_8_18_4 = def(8_840_0_06);
-    public static final TransportVersion INITIAL_ELASTICSEARCH_8_18_6 = def(8_840_0_08);
+    public static final TransportVersion INITIAL_ELASTICSEARCH_8_18_5 = def(8_840_0_07);
```

Comment:
is there a reason we need to add this back? I know the point is to get us back to density, but moving the upper bound forward to the 8 patch doesn't seem like it requires unconverting the initial version right before it?

Reply:
I can add these back in.